target/linux/rockchip/armv8/base-files/etc/board.d/02_network

```diff
@@ -60,18 +40,19 @@ rockchip_setup_macs()
 		wan_mac=$(macaddr_generate_from_mmc_cid mmcblk0)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
-	friendlyarm,nanopi-r4s)
-		wan_mac=$(nanopi_r4s_get_mac wan)
-		lan_mac=$(nanopi_r4s_get_mac lan)
+	friendlyarm,nanopi-r4s|\
+	friendlyarm,nanopi-r5s)
+		wan_mac=$(macaddr_generate_from_mmc_cid mmcblk1)
+		lan_mac=$(macaddr_add "$wan_mac" 1)
+		;;
+	friendlyarm,nanopi-r4s-enterprise)
+		wan_mac=$(get_mac_binary "/sys/bus/i2c/devices/2-0051/eeprom" 0xfa)
+		lan_mac=$(macaddr_setbit_la "$wan_mac")
 		;;
 	friendlyarm,nanopi-r5c)
 		wan_mac=$(macaddr_generate_from_mmc_cid mmcblk*)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
-	friendlyarm,nanopi-r5s)
```

Comment:
Why is the entry for nanopi-r5s removed here?  Your comment makes no mention of this device.

Reply:
https://github.com/openwrt/openwrt/pull/14657/commits/f9c254c33a723f810f18bb8d0bd770a639a820cf#diff-1ce315e56a6173a0798c249e226c43413ee01c9b5e941f56886d938001d3228fR44